src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchShareLogDAO.scala

```diff
@@ -13,6 +14,7 @@ import org.elasticsearch.action.support.WriteRequest.RefreshPolicy
 import org.elasticsearch.client.transport.TransportClient
 import org.elasticsearch.common.xcontent.XContentType
 import spray.json._
+import spray.json.DefaultJsonProtocol._
```

Comment:
needed to handle json de/serialization of `List[T]`, which in this case is `List[Share]`